### ISSUE TITLE
carrier broadcast syntax

### ISSUE THREAD

**Issue by HILOK** [rkusa/DATIS#96]
is there another way than renaming the carrier unit in order to provide a carrier based wx broadcast? -problem is in dcs_liberation every unit has a specific name to it that should better not be changed, i guess...
thanks : )

**Comment by rkusa** [comment]
Good idea, I'll keep it in mind for the next times I am working on DATIS 👍

**Comment by HILOK** [comment]
that would be great! i hope DATIS could find its way into liberation some day : )
thanks
